src/flask_sqlalchemy/query.py

```diff
@@ -2,16 +2,15 @@
 
 import typing as t
 
-import sqlalchemy as sa
-import sqlalchemy.exc
-import sqlalchemy.orm
+import sqlalchemy.exc as sa_exc
+import sqlalchemy.orm as sa_orm
 from flask import abort
 
 from .pagination import Pagination
 from .pagination import QueryPagination
 
 
-class Query(sa.orm.Query):  # type: ignore[type-arg]
+class Query(sa_orm.Query[t.Any]):
```

Comment:
This seems correct per https://github.com/sqlalchemy/sqlalchemy/blob/1c3dbf39ec2d72ea70c0f74fa2e6dc537e76700b/lib/sqlalchemy/orm/query.py#L151

Reply:
According to this, https://github.com/sqlalchemy/sqlalchemy/blob/4808acfe1b88854fce47b792ef9995f1921426e7/lib/sqlalchemy/orm/query.py#L77
which is the version(`1.4.18`) specified in the `requirements/tests-min.in`, it's not correct. I will fix it.